lib/net/networking_engine.js

```diff
@@ -46,7 +46,8 @@ goog.require('shaka.util.Timer');
  */
 shaka.net.NetworkingEngine = class extends shaka.util.FakeEventTarget {
   /**
-   * @param {function(number, number, boolean)=} onProgressUpdated Called when
+   * @param {shaka.net.NetworkingEngine.onProgressUpdated=
+   *   } onProgressUpdated Called when
```

Comment:
The formatting looks weird.  Can you move this to the previous line?  Or will the linter/compiler not accept it?